modules/javafx.web/src/test/java/test/javafx/scene/web/LoadNotificationsTest.java

```diff
@@ -147,11 +146,6 @@ private void checkState(State oldValue, State newValue) {
                     assertNotNull("WebEngine.document should be set", web.getDocument());
                     assertTrue("WebEngine.location should be set",
                             web.getLocation().endsWith(currentUrl));
-                    if (currentUrl == "about:blank") {
-                        assertNull("WebEngine.title should be null", web.getTitle());
-                    } else {
-                        assertNotNull("WebEngine.title should be set", web.getTitle());
```

Comment:
Similar to the comment I made in `HistoryTest`, is it worth filing a P4 test bug to look at converting this to use a listener? If so, then leaving this in, but commented out might be a good idea.

Reply:
I have reverted this change and commented the code and created [JDK-8269912](https://bugs.openjdk.java.net/browse/JDK-8269912) to investigate test.